### ISSUE TITLE
modifier combination of skipws and ws produces weird result

### ISSUE THREAD

**Issue by wangluochao902** [textX/textX#246]
Here is a grammer to reproduce:
```
from textx import metamodel_from_str
from textx.exceptions import TextXSyntaxError

meta_s = '''
main_rule[ws=' \t']:
    "word1" rule1
;

rule1[skipws, ws=' \t\n']:
    "word2" "word3"
;
'''

meta = metamodel_from_str(meta_s)
s = '''word1 word2
word3'''

model = meta.model_from_str(s)

```
Here is the error message:
```
textx.exceptions.TextXSyntaxError: None:1:12: error: Expected 'word3' at position (1, 12) => 'ord1 word2* word3'.
```

If I delete `skipws` from rule1. Then every thing is good. I know `skipws` is a default option. This  example is just a simplified version of my complex grammer.

**Comment by igordejanovic** [comment]
Looks like a bug. Thanks for reporting.

**Comment by igordejanovic** [comment]
@wangluochao902 The fix is available on #251 . Please verify.

**Comment by wangluochao902** [comment]
Thanks for fixing it. It works~
